tests/source/extern.rs

```diff
@@ -63,3 +63,20 @@ libc::c_long;
 extern {
 
 }
+
+// #2908 - https://github.com/rust-lang/rustfmt/issues/2908
+#[wasm_bindgen(module = "child_process", version = "*")]
+extern {
+     #[wasm_bindgen(js_name = execSync)]
+     fn exec_sync(cmd: &str) -> Buffer;
+
+     fn foo() -> Bar;
+ }
+
+// Users that have an existing explicit ABI would need to convert to implicit
+// manually, as rustfmt will be conservative and not attempt to convert explicit
+// to implicit in the wasm case.
+#[wasm_bindgen]
+extern "C" {
+       fn foo() -> Bar;
+}
```

Comment:
Unfortunately I believe that an explicit C ABI became common practice with wasm_bindgen at least in part because of rustfmt 1.x behavior. However, I don't think we should attempt to help those users convert _back_ to an implicit ABI as it seems to risky/error prone IMO; with this change, rustfmt 2.0 by default would permit users to do what they want and will no longer force Explicit ABIs.